camel-core/src/main/java/org/apache/camel/ComponentVerifier.java

```diff
@@ -412,6 +411,12 @@ public String toString() {
         }
     }
 
+    class ExceptionAttributeImpl extends ErrorAttribute implements VerificationError.ExceptionAttribute {
+        private ExceptionAttributeImpl(String name) {
+            super(name);
+        }
+    }
+
```

Comment:
I think we can keep the ExceptionAttributeImpl in the interface for the moment as it does not change how you consume the ComponentVerifier interface and refactor it later if we'll find a better solution